packages/hash/api/src/db/postgres/aggregation/getAggregation.ts

```diff
@@ -0,0 +1,24 @@
+import { sql } from "slonik";
+import { DbAggregation, DbClient } from "../../adapter";
+
+import { Connection } from "../types";
+import {
+  DbAggregationRow,
+  aggregationsColumnNamesSQL,
+  mapRowToDbAggregation,
+} from "./util";
+
+export const getAggregation = async (
+  conn: Connection,
+  params: Parameters<DbClient["getAggregation"]>[0],
+): Promise<DbAggregation | null> => {
+  const row = await conn.maybeOne<DbAggregationRow>(sql`
+    select ${aggregationsColumnNamesSQL}
+    from aggregations
+    where aggregation_id = ${params.aggregationId}
+    order by created_at desc
+    limit 1 -- @todo: remove when aggregation versions are stored in separate table
```

Comment:
Just so I understand correctly regarding this todo, would having aggregation version not mean that there would be a separate table where with the versions, changing this query entirely?

Reply:
Yes, although this todo is only relevant to `getEntityAggregationByPath` I'm realising. We don't need `order by ... limit 1` for this query as the `aggregation_id` is unique in the table. Have changes that in this commit: https://github.com/hashintel/hash/pull/465/commits/08ea61df85c98fc61fcb09c05ad2b4406b10b3e4